### ISSUE TITLE
训练结果全部变成<GO><GO><GO>

### ISSUE THREAD

**Issue by l00350898** [wushilian/CRNN_Attention_OCR#25]
你好，使用模型训练，项目中的训练集，当loss=0.18，精确度71%，learning_rate由0.001调到0.0001的时候，模型突然所有的预测结果都变成一连串的<GO><GO><GO>....<GO>
请问一下，这个是什么原因导致的？谢谢啦

**Comment by wushilian** [comment]
@l00350898 just turn the learning rate to 1e-5
